docs/index.md

```diff
@@ -1,37 +1,37 @@
-* [Animations notebook](https://github.com/riveSunder/yuca/blob/master/notebooks/glaberish_animations.ipynb) Try it on: [mybinder](https://mybinder.org/v2/gh/rivesunder/yuca/master?labpath=notebooks%2Fglaberish_animations.ipynb) -> [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/rivesunder/yuca/master?labpath=notebooks%2Fglaberish_animations.ipynb), [colab](https://colab.research.google.com/github/rivesunder/yuca/blob/master/notebooks/glaberish_animations.ipynb) -> [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rivesunder/yuca/blob/master/notebooks/glaberish_animations.ipynb) 
-* [ALife 2022 slides](https://rivesunder.github.io/yuca/g_slide_000)
+* [Supporting animations](https://rivesunder.github.io/yuca_docs/glaberish) 
+* [Static figures notebook](https://github.com/riveSunder/yuca_docs/blob/master/notebooks/glaberish_static_figures.ipynb) Try it on: [mybinder](https://mybinder.org/v2/gh/rivesunder/yuca_docs/master?labpath=notebooks%2Fglaberish_static_figures.ipynb) -> [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/rivesunder/yuca_docs/master?labpath=notebooks%2Fglaberish_static_figures.ipynb), [colab](https://colab.research.google.com/github/rivesunder/yuca_docs/blob/master/notebooks/glaberish_static_figures.ipynb) -> [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rivesunder/yuca_docs/blob/master/notebooks/glaberish_static_figures.ipynb) 
+* [Animations notebook](https://github.com/riveSunder/yuca_docs/blob/master/notebooks/glaberish_animations.ipynb) Try it on: [mybinder](https://mybinder.org/v2/gh/rivesunder/yuca_docs/master?labpath=notebooks%2Fglaberish_animations.ipynb) -> [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/rivesunder/yuca_docs/master?labpath=notebooks%2Fglaberish_animations.ipynb), [colab](https://colab.research.google.com/github/rivesunder/yuca_docs/blob/master/notebooks/glaberish_animations.ipynb) -> [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rivesunder/yuca_docs/blob/master/notebooks/glaberish_animations.ipynb) 
+* [ALife 2022 slides](https://rivesunder.github.io/yuca_docs/g_slide_000)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.